src/components/MyTips/MyTips.tsx

```diff
@@ -19,12 +19,17 @@ const Tip = ({ tip }: TipProps) => (
       {tip.imgUrl ? (
         <img src={tip.imgUrl} />
       ) : (
-        <ImgPlaceholder className={styles.ImgPlaceholder} />
+        <ImgPlaceholder aria-hidden="true" className={styles.ImgPlaceholder} />
       )}
     </div>
     <Heading el="h4">{tip.title}</Heading>
     <p>{tip.description}</p>
-    <ButtonLinkExternal to={tip.link.to}>{tip.link.title}</ButtonLinkExternal>
+    <ButtonLinkExternal
+      aria-label="Meer informatie over deze tip"
```

Comment:
Same here, the link.title should be clarified instead of label added.

Reply:
Done